### ISSUE TITLE
Showing only bookmark icon in bar

### ISSUE THREAD

**Issue by Reprevise** [brave/brave-browser#9141]
## Description 

Brave should allow users to be able to show icon only for bookmarks in the bookmarks bar by right-clicking on the desired bookmark. Microsoft Edge already has this feature and it would be great to do so in Brave without having to remove the name of the bookmark so you can see it in the bookmark manager.

## Actual result:

(When you right click a bookmark currently)
![image](https://user-images.githubusercontent.com/25782944/79024976-7f35fa00-7b52-11ea-8d5c-b7bad7c823aa.png)

## Expected result:
There should be an option in the dropdown that reads "Show icon only" or something similar that removes the label.

(Here's what it looks like on Microsoft Edge)
![image](https://user-images.githubusercontent.com/25782944/79025140-ece22600-7b52-11ea-8259-977946d3d000.png)

## Brave version (brave://version info)

**Brave**: 1.5.123 Chromium: 80.0.3987.163 (Official Build) (64-bit)
**Revision**: e7fbe071abe9328cdce4ffedac9822435fbd3656-refs/branch-heads/3987@{#1037}
**OS**: Windows 10 OS Version 1909 (Build 18363.752)

## Version/Channel Information:

- Can you reproduce this issue with the current release? Yes
- Can you reproduce this issue with the beta channel? Unknown
- Can you reproduce this issue with the dev channel? Unknown
- Can you reproduce this issue with the nightly channel? Unknown

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? N/A
- Does the issue resolve itself when disabling Brave Rewards? N/A
- Is the issue reproducible on the latest version of Chrome? It's not a feature on Chrome.

## Miscellaneous Information:

Nope.

**Comment by rebron** [comment]
@Reprevise There is a workaround now that if you edit and delete the title, you can show just the favicon. On hover, you should still be able to see the url.

**Comment by Reprevise** [comment]
Yes, I mentioned that in my issue. However, in the bookmark manager, it will just show the favicon and unless you know the logo, you don't know the site.
![image](https://user-images.githubusercontent.com/25782944/79026350-8828ca80-7b56-11ea-91e7-f9ba5249db24.png)

**Comment by dpthegrey** [comment]
yeah editing and removing title will only show the logo but if we have multiple url of same site bookmarked, we will see icons and url but no title. I would love to have this option to only show logo while preserving the title.

**Comment by appel** [comment]
Not to mention that if the title was preserved you could have a title tooltip on icon hover, plus searching might be more efficient in the bookmarks manager.
